### PR TITLE
Adds stylemedia back

### PR DESCRIPTION
**Pull request by orta**
`StyleMedia` is used in React's types

### CODE REVIEW

**Comment by saschanaz** [comment]
I'm not sure why it's used and for what. It doesn't exist anywhere. If it only uses the type but not the value, maybe `noInterfaceObject: true`?

**Comment by saschanaz** [comment]
Just took a look at React types. It's used in `interface AbstractView` which seems to be mocking `window` with only two properties. Since `styleMedia` only existed in IE11 and nowhere else, it should not exist there. And it only uses the type, so I guess https://github.com/saschanaz/types-web/pull/113#issuecomment-867681386 should be acceptable.

**Comment by orta** [comment]
OK, this is rebased, commented and updated with feedback

File: inputfiles/addedTypes.json [hunk 1]
```diff
@@ -616,6 +616,32 @@
...
+            "StyleMedia": {
+                "name": "StyleMedia",
+                "exposed": "Window",
+                "noInterfaceObject": true,
+                "deprecated": true,
+                "properties": {
+                    "property": {
+                        "type": {
+                            "name": "type",
+                            "overrideType": "string"
```
Comment (saschanaz) [inline inputfiles/addedTypes.json:630]:
```suggestion
                            "type": "DOMString"
```

**Comment by saschanaz** [comment]
Thanks!

**Pull request merged**